experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -244,37 +255,20 @@ end
 
 Y.canopy.energy.T = drivers.TA.values[1 + Int(round(t0 / DATA_DT))] # Get atmos temperature at t0
 
+Y.snow.S .= 0.0
+Y.snow.U .= 0.0
+
 set_initial_cache! = make_set_initial_cache(land)
 set_initial_cache!(p, Y, t0);
 
-# Simulation
```

Comment:
no longer using the SavingCallback here (using Diagnostics). This was leftover from a previous PR so I just removed it here!